lib/sufia/arkivo/schema_validator.rb

```diff
@@ -0,0 +1,57 @@
+require 'json-schema'
+
+module Sufia
+  module Arkivo
+    ITEM_SCHEMA = {
+      type: 'object',
+      properties: {
+        token: { type: 'string', required: true },
+        metadata: {
+          type: 'object',
+          required: true,
+          properties: {
+            title: { type: 'string', required: true },
+            creators: { type: 'array', required: true },
+            rights: { type: 'string', required: true },
+            tags: { type: 'array', required: true },
```

Comment:
Is there a way to specify type is 'array' or 'null'? Just setting required to false did not fix it for me, because Rails actually inserts a null value there.